### PR TITLE
create a sample using typing indicator for group channel

### PR DESCRIPTION
**Pull request by bogdanmihaila**

### CODE REVIEW

File: src/samples/GroupChannelTypingIndicatorSample.js [hunk 1]
```diff
@@ -0,0 +1,647 @@
...
+
+    if (state.loading) {
+        return <div>Loading...</div>
+    }
+
+    if (state.error) {
+        return <div className="error">{state.error} check console for more information.</div>
+    }
+
+    const displayTypingIndicator = () => {
```
Comment (james-a-rob) [inline src/samples/GroupChannelTypingIndicatorSample.js:271]: Could we please move this component down to to where other components are defined. starting line 334. Nice to keep that pattern going so users know where to look

Comment (james-a-rob) [reply src/samples/GroupChannelTypingIndicatorSample.js:271]: Also capital for react components please. DisplayTypingIndicator. 🙏🏻

**Comment by octa-george** [comment]
anything else to add here @james-a-rob ?

**Comment by AAPK-code** [comment]
@james-a-rob PR updated

**Review by james-a-rob — APPROVED** [review]

**Pull request merged**